docs/getting_started_guide/Installing_SIMP_From_An_ISO.rst

```diff
@@ -0,0 +1,32 @@
+.. _gsg-installing_simp_from_an_iso:
+
+Installing SIMP From An ISO
+===========================
+
+The benefits of using a SIMP ISO are:
```

Comment:
I think this section should go in a parent 'How to install guide' section that would help the user choose which installation guide they want to follow. This may be done in another ticket tho